acra/configuring-maintaining/general-configuration/acra-rotate.md

```diff
@@ -30,12 +30,14 @@ weight: 10
 
 * `--sql_select=<query>`
 
-  Select query with `?` as placeholders where last columns in result must be ClientId/ZoneId and AcraStruct. 
-  Other columns will be passed into insert/update query into placeholders.
+  Select query with placeholders where last columns in result must be ClientId/ZoneId and AcraStruct. 
+  Other columns will be passed into Insert/Update query into placeholders.
+  PostgreSQL placeholder is `$1` and MySQL is `?`.
 
 * `--sql_update=<query>`
 
-  Insert/Update query with `?` as placeholder where into first will be placed rotated AcraStruct.
+  Insert/Update query with placeholders where rotated AcraStruct will be placed into the first.
+  PostgreSQL placeholder is `$1` and MySQL is `?`.
```

Comment:
actually, not `$1` but `$n`. And maybe will be better to add references to source of such requirements: db drivers.
https://github.com/lib/pq - driver responsible for postgresql query execution and https://github.com/go-sql-driver/mysql for MySQL queries.